src/OpenIddict.Abstractions/Resources/OpenIddictResources.resx

```diff
@@ -1385,6 +1385,9 @@ To register the OpenIddict core services, reference the 'OpenIddict.Core' packag
     <value>The implicit flow must be enabled when adding a response type containing '{0}'.</value>
     <comment>{Locked}</comment>
   </data>
+  <data name="ID0283" xml:space="preserve">
+    <value>Provided symmetric key was incorrect size. Must be 256 bits derived from exactly 32 bytes or string of length 32.</value>
```

Comment:
Can you please make it parametrized? Also, I'd remove the `derived from exactly 32 bytes or string of length 32` part, which is a bit confusing.